### ISSUE TITLE
Question: Fallback font if one isn't set?

### ISSUE THREAD

**Issue by hahuang65** [alacritty/alacritty#3573]
### System

OS: Arch Linux
Version: alacritty 0.4.2
Linux/BSD: X11, i3

Attached is my full alacritty config.

[alacritty.yml.txt](https://github.com/alacritty/alacritty/files/4460725/alacritty.yml.txt)

I'm trying to figure out, why certain font glyphs are rendering if the font I have selected does not support them.

Here is the link to the issue for repo for the font I'm using: https://github.com/JetBrains/JetBrainsMono/issues/203

Basically, they don't support glyphs I'm seeing in my terminal, yet I don't have any fallback fonts configured. What could cause these glyphs to render?

**Comment by kchibisov** [comment]
fallback font is picked based off the output of `fc-match -s YOUR_FONT:style=YOUR_FONT_STYLE:pixelsize=YOUR_FONT_SIZE_IN_PX`. we're going throughout the entire output you'll see and pick fonts when glyphs are missing.

**Comment by hahuang65** [comment]
Thank you for the really fast response! It was really helpful, thank you!
